### PR TITLE
App settings support

### PR DESCRIPTION
**Pull request by dgarciarubio**
I was required to configure these enrichers from app settings for a project, so I decided to add a PR to share my work.

Related issue [#34](https://github.com/serilog-web/classic/issues/34)

### CODE REVIEW

File: src/SerilogWeb.Classic/SerilogWebClassicLoggerConfigurationExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        /// <summary>
+        /// Enrich log events with the Url of the Request.
+        /// </summary>
+        /// <param name="enrichmentConfiguration">Logger enrichment configuration.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public static LoggerConfiguration WithHttpRequestUrl(
+            this LoggerEnrichmentConfiguration enrichmentConfiguration)
+        {
+            if (enrichmentConfiguration == null) throw new ArgumentNullException(nameof(enrichmentConfiguration));       
+            return enrichmentConfiguration.With< HttpRequestUrlEnricher>();
```
Comment (tsimbalar) [inline src/SerilogWeb.Classic/SerilogWebClassicLoggerConfigurationExtensions.cs:93]: nitpicking : there is a whitespace between `With<` and `HttpRequestUrlEnricher` :)

**Comment by tsimbalar** [comment]
Hi Daniel,

Thanks for your PR, it looks pretty good to me !

If you have time, it would also be good to update the README file (in the [*Enrichers*](https://github.com/serilog-web/classic#enrichers) section : 
- use `WithXXX()` instead of the the *enricher's* class name in the list of provided *Enrichers* 
- use the `WithXXX()` method in the code sample
- to at least mention that the enrichers can be defined also in config file with *Serilog.Settings.AppSettings* and put an example (taking the `appSettings` section from ` test/SerilogWeb.Test/Web.config` is probably good enough)

When this is done, I'll increment the version number and publish a new version.

Thanks a lot !

**Comment by dgarciarubio** [comment]
I will probably be able to do so this week. I was also not sure wether the extensions class should go inside the Classic folder or not... I guess it is fine outside...?

**Comment by tsimbalar** [comment]
Yes, that one is a bit special, we do want it in the `Serilog` namespace as you did it, and our root namespace is `SerilogWeb`, so there's no ideal place. 

I think your choice is just fine, and the `Classic` folder is already pretty crowded ;)

**Comment by tsimbalar** [comment]
Awesome ! 

Here is the new version on Nuget : https://www.nuget.org/packages/SerilogWeb.Classic/4.2.41 

Congratulations on your first merged PR ! 🎉

**Comment by tsimbalar** [comment]
If you feel like adding the same kind of improvements in *SerilogWeb.Classic.Mvc* and *SerilogWeb.Classic.WebApi* , help is more than welcome ;) 
(see serilog-web/classic-mvc#8 and serilog-web/classic-webapi#14 )

**Pull request merged**